plumgrid_playbooks/roles/plumgrid_common/tasks/main.yml

```diff
@@ -45,51 +40,37 @@
     file: /tmp/GPG-KEY
     state: present
 
-# Update repositories
-- name: Running apt-update
```

Comment:
@JaveriaK we don't need it seperately, as you can  on the line [46](https://github.com/plumgrid/plumgrid-ansible-standalone/pull/1/files#diff-16f9a04530fb591ecea22ad736daa7cbR46) I have added the `update_cache: yes` which serves the same purpose.

```yaml
- name: Install iovisor and plumgrid-lxc
   apt:
       name: "{{ item }}" 
       update_cache: yes
 ```
Please let me know if you have any question. Thanks